MainGame.py

```diff
@@ -15,7 +16,7 @@ def PrepareGame(self) -> None:
         self.playerTwo.name = self.GetUserName("Spieler 2")
         self.playerTwo.symbol = "O"
 
-    def Turn(self, player: Player) -> None:
+    def Turn(self, player: Player) -> None or Player:
```

Comment:
`None or Player` is not a valid type :) Use [Optional](https://docs.python.org/3/library/typing.html#typing.Optional) or [Unions](https://docs.python.org/3/library/typing.html#typing.Union) instead